src/DI/CzechPostExtension.php

```diff
@@ -17,10 +19,12 @@ class CzechPostExtension extends CompilerExtension
 	/** @var mixed[] */
 	protected $defaults = [
 		'http' => [
-			'base_uri' => null,
-			'auth' => [],
+			'base_uri' => 'https://online.postservis.cz/',
+			'auth' => ['user', 'pass'],
+		],
+		'config' => [
+			'tmp_dir' => 'path/to/tmp/dir',
```

Comment:
Proc jsou tu defaultni hodnoty?

Reply:
self dokumentace. prijde mi lepsi tam mit toto nez 'auth' => ['', ''] aby clovek musel premyslet co jaky parametr je

Reply:
Kdyz na to uzivatel bude mit prava, tak mu to vytvori na tehle ceste soubory? To mi neprijde moc dobry. Tohle patri do dokumentace.

Reply:
Realne kazdej tu knihovnu nejdriv zkusi nez cte zdrojaky.

Reply:
odstraneno